common/src/test/java/com/sequenceiq/cloudbreak/auth/crn/CrnTest.java

```diff
@@ -4,16 +4,12 @@
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
```

Comment:
Should use Jupiter `Assertions`.
Same for all other test files.

Reply:
Done, in every test class under common module.